### ISSUE TITLE
Use READMD.md to write index.rst in documentation

### ISSUE THREAD

**Issue by bobleesj** [diffpy/diffpy.nmf_mapping#11]
The original `README.md` before cookiecutting has lots of useful information while there is no `index.rst` provided in the documentation. 

We shall use the information to write the `index.rst` file.

**Comment by sbillinge** [comment]
yes, please do take care of blending the info from the old EADME.md into the new README.rst, and also as you mention, this can be reused in the docs and the long_description in pypi/conda-forge
